modules/understanding-upgrade-channels.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// updating/updating-cluster.adoc
+// updating/updating-cluster-cli.adoc
+// updating/updating-cluster-rhel-compute.adoc
+// updating/updating-disconnected-cluster.adoc
+
+[id="understanding_upgrade_channels_{context}"]
+= Understanding {product-title} upgrade channels
+
+In {product-title} 4.1, Red Hat introduced the concept of upgrade channels for
+recommending the appropriate upgrade versions to your cluster. Upgrade channels
+separate upgrade strategies and also are used to control the cadence of updates.
+Channels are tied to a minor version of {product-title}. For instance,
+{product-title} {product-version} channels will never include an upgrade to a
+4.3 release. This ensures administrators make an explicit decision to upgrade to
+the next minor version of {product-title}. Channels only control updates and
+have no impact on the version of the cluster you install; the
+`openshift-install` binary for a given patch level of {product-title} always
+installs that patch level.
+
+See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades
```

Comment:
Is there a reason we didn't include the full description?  Why go to an article?

Reply:
@smarterclayton From the discussions I was following, it sounded like the content could continue to evolve and that we could potentially add in more types. I didn't want to duplicate any information and effort and thought pointing to the article could help with future docs maintenance while that is all sorted out. Let me know if that's a misunderstanding on my part. If so, we can pull the content into our doc set instead. Thanks!

Reply:
That's fine, we can always do it later.  I would like to see some of the discussion in the doc in the long run, but yes, it may still be evolving a bit.

Reply:
@smarterclayton, is that KB stable enough to roll into the docs next week?